style.css

```diff
@@ -112,3 +112,9 @@ a {
 a:hover {
   opacity: 0.7;
 }
+
+.clearfix::after {
+  display: table;
```

Comment:
Can you explain why you did choose `display: table`, do you think we can achieve the same results using `display: block` ?